subsys/bluetooth/host/conn.c

```diff
@@ -512,8 +535,12 @@ static int send_iso(struct bt_conn *conn, struct net_buf *buf, uint8_t flags)
 	}
 
 	hdr = net_buf_push(buf, sizeof(*hdr));
-	hdr->handle = sys_cpu_to_le16(bt_iso_handle_pack(conn->handle, flags,
-							 0));
+
+	ts = tx_data(buf)->iso_has_ts &&
+		(flags == BT_ISO_START || flags == BT_ISO_SINGLE);
```

Comment:
Is there any spec requirements that state that we cannot send the TS to the controller for e.g. `BT_ISO_CONT`?

Reply:
yes
![image](https://user-images.githubusercontent.com/68428270/235646627-8e4267f7-58d5-4fb5-9a5a-6ce090b5424a.png)